src/@types/BarChart.tsx

```diff
@@ -21,20 +21,13 @@ export interface ChartDataset {
 }
 
 export interface CustomCalendarProps {
-  title?: string;
-  placeholder?: string;
-  name?: string;
-  timeOnly?: boolean;
-  showTime?: boolean;
-  hideOnDateTimeSelect?: boolean;
-  hourFormat?: "24" | "12";
-  selectionMode?: "single" | "range" | "multiple";
-  value: string;
+  title: string;
+  placeholder: string;
+  value: Date;
   onChange: (value: Date) => void;
-  iconPos: "left" | "right";
```

Comment:
Recheck the lines that u removed here.
I had them for a reason. ex: containerClassName